README.md

```diff
@@ -124,6 +124,7 @@ Downloads the RHCOS initrd with the ignition for the specified image appended.
 
 - `version`: indicates the version of the RHCOS base image to use (must match an entry in `RHCOS_VERSIONS`)
 - `arch`: the base image cpu architecture (must match an entry in `RHCOS_VERSIONS`)
+- `addrsize`: indicates if the initrd.addrsize (`1`) file instead of initrd.img (`0`) should be returned. Only valid for s390x archtiecture
```

Comment:
@paul-maidment Thank you for bringing this up. Maybe I bring some light into the thoughts from s390x community. Values like 0 and 1 are very common. In boot files like parm files values rd.neednet=1 or zfcp.allow_lun_scan=0 are used. Therefor I choose the value that way. But I understand that for users working with different architectures would get confused.

So what if we do it that way:
- change the parameter from 'addrsize' to 'file'
- value initrd.img or initrrd.addrsize with default initrd.img

Is this a feasible way to go?

Reply:
It should just be a boolean. true or false

Reply:
See below suggestion for my recommended phrasing. I mostly didn't like the use of "returned" since it's a too generic word and doesn't give much context of when and what

Reply:
>from s390x community. Values like 0 and 1 are very common

This is the semantics of our HTTP service API, the conventions of the s390x community / kernel boot arguments don't apply